ckan/config/middleware/flask_app.py

```diff
@@ -394,10 +395,23 @@ def ckan_after_request(response: Response) -> Response:
     # Set Cache Control headers
     response = set_cache_control_headers_for_response(response)
 
+    # Set ETag headers and unchanged response if config allows it
+    response = set_etag_and_fast_304_response_if_unchanged(response)
+
     r_time = time.time() - g.__timer
     url = request.environ['PATH_INFO']
     status_code = response.status_code
 
+    is_sensitive = u'__is_sensitive__' in request.environ
```

Comment:
`__dunder__` names indicate "has internal python meaning" which doesn't apply here.  Something like `'cache_middleware_is_sensitive'` would give users a better idea of what this variable is intended for.